recipes/oh-my-posh/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+pushd src
+    go build -buildmode=pie -trimpath -o=${PREFIX}/bin/${PKG_NAME} -ldflags="-s -w -X main.Version=${PKG_VERSION}"
+    go-licenses save . --save_path=../license-files \
+        --ignore github.com/golang/freetype/raster \
+        --ignore github.com/golang/freetype/truetype \
+        --ignore github.com/jandedobbeleer/oh-my-posh \
```

Comment:
Same as in the other PR: please add these manually.

Reply:
All of these have a license which I will add.  The only one that doesn't need to be manually added is github.com/jandedobbeleer/oh-my-posh, because that is the repository for this tool, for which the main `LICENSE` file is already being added.